vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -188,7 +268,25 @@ public void deleteImageBlockById(String id) throws BlockDoesNotExistException {
         }
 
     }
+    /**
+     * Delete an video block using an id
+     * 
+     * @param id - id of resource to be deleted. If the id is null then the
+     *           functions returns nothing.
```

Comment:
this is a method (not a function) and it never returns anything (its return type is `void`)